ubuntu/common/install_intel_libs.sh

```diff
@@ -1,21 +1,10 @@
 #!/bin/bash
 set -ex
 
-# Intel® oneAPI Math Kernel Library
-case ${DISTRIBUTION} in
-    "ubuntu18.04") INTEL_MKL_VERSION="2023.1.0.46342";
-        RELEASE_VERSION="cd17b7fe-500e-4305-a89b-bd5b42bfd9f8";
-        CHECKSUM="cc28c94cab23c185520b93c5a04f3979d8da6b4c90cee8c0681dd89819d76167";; 
-    "ubuntu20.04") INTEL_MKL_VERSION="2023.1.0.46342";
-        RELEASE_VERSION="cd17b7fe-500e-4305-a89b-bd5b42bfd9f8";
-        CHECKSUM="cc28c94cab23c185520b93c5a04f3979d8da6b4c90cee8c0681dd89819d76167";;
-    "ubuntu22.04") INTEL_MKL_VERSION="2023.1.0.46342";
-        RELEASE_VERSION="cd17b7fe-500e-4305-a89b-bd5b42bfd9f8";
-        CHECKSUM="cc28c94cab23c185520b93c5a04f3979d8da6b4c90cee8c0681dd89819d76167";;
-    *) ;;
-esac
+# Set Intel® oneAPI Math Kernel Library version
+intel_one_mkl_version=$(jq -r '.intel_one_mkl."'"$DISTRIBUTION"'".version' <<< $COMPONENT_VERSIONS)
 
-ONE_MKL_DOWNLOAD_URL=https://registrationcenter-download.intel.com/akdlm/IRC_NAS/${RELEASE_VERSION}/l_onemkl_p_${INTEL_MKL_VERSION}_offline.sh
-$COMMON_DIR/write_component_version.sh "INTEL_ONE_MKL" ${INTEL_MKL_VERSION}
-$COMMON_DIR/download_and_verify.sh ${ONE_MKL_DOWNLOAD_URL} ${CHECKSUM}
-sh ./l_onemkl_p_${INTEL_MKL_VERSION}_offline.sh -s -a -s --eula accept
+spack add intel-oneapi-mkl@$intel_one_mkl_version
+# If there is a space crunch use spack gc (this performs garbage collection)
```

Comment:
Any guidance that can be added to this comment? (i.e. what counts as a space crunch?)

Reply:
Added command for garbage collection